extensions/wrapper/wrapper-api/src/main/java/de/sovity/edc/ext/wrapper/api/ui/model/ContractAgreementPageQuery.java

```diff
@@ -26,5 +26,5 @@ public class ContractAgreementPageQuery {
     @Schema(
         description = "Optionally filter the resulting contract agreements by their termination status.",
         requiredMode = Schema.RequiredMode.NOT_REQUIRED)
-    private ContractTerminationStatus terminationStatus;
+    private ContractTerminationStatus status;
```

Comment:
is this meant to be part of the PR?

Reply:
No, I though I fixed it in the review...